wiki/MapData.md

```diff
@@ -537,7 +537,7 @@ security <security#>
 
 The behavior that this planet has when the player has illegal goods or outfits. The bribe number is a multiplier that modifies the severity of any fines, while the security number is a value between 0 and 1 that dictates the chance of a player's ship being scanned.
 
-If no security is specified, then a default security of 0.25 is used. 
+If no bribe is specified, then a default bribe of 0.01 is used. If no security is specified, then it defaults to 0.25. 
```

Comment:
I think this would be better if the two items are on their own lines.
I also think they should be formatted the same:
```suggestion
If no bribe is specified, then it defaults to 0.01.
If no security is specified, then it defaults to 0.25. 
```
Or:
```suggestion
If no bribe is specified, then a default bribe of 0.01 is used.
If no security is specified, then a default security of 0.25 is used. 
```

Reply:
Okay. I'll commit the second. It sounds a little stilted, though, if they're all formatted the same..